creator-node/src/services/prometheusMonitoring/prometheus.constants.js

```diff
@@ -0,0 +1,56 @@
+const promClient = require('prom-client')
+const _ = require('lodash')
+
+/**
+ * For explanation of Metrics, and instructions on how to add a new metric, please see `prometheusMonitoring/README.md`
+ */
+
+// We add a namespace prefix to differentiate internal metrics from those exported by different exporters from the same host
+const AudiusPrefix = 'audius_'
+const DefaultPrefix = 'default_'
+
+/**
+ * @notice Counter and Summary metric types are currently disabled, see README for details.
+ */
+const MetricTypes = Object.freeze({
+  GAUGE: promClient.Gauge,
+  HISTOGRAM: promClient.Histogram
+  // COUNTER: promClient.Counter,
+  // SUMMARY: promClient.Summary
+})
+
+let MetricNames = {
+  SYNC_QUEUE_JOB_COUNTS_GAUGE: 'sync_queue_job_counts',
```

Comment:
`_total` according to the [standard naming scheme](https://prometheus.io/docs/practices/naming/).

Reply:
good call, fixed